src/Education/Education.csproj

```diff
@@ -0,0 +1,9 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net4.7.2</TargetFramework>
```

Comment:
We also have a common product properties file you can include, so you don't need to set the target framework explicitly.

```
  <Import Project="..\..\SonarLint.Product.props" />
```